tools/ci_build/github/azure-pipelines/win-gpu-ci-pipeline.yml

```diff
@@ -1,3 +1,15 @@
+trigger:
+  branches:
```

Comment:
It looks like you only set the policy for "continuous integration", but not "pull request validation" which we care most.